schedule/functional/allmodules+allpatterns-gnome.yaml

```diff
@@ -1,21 +1,12 @@
 ---
 name: allmodules+allpatterns-gnome
 description: >
-    Maintainer: zluo
-    Installation with all patterns and all modules without post-installation
-    test modules and it doesn't publish qcow2 image. Exclude modules not
-    designed to work with all other modules together
+    Maintainer: foursixnine.
+    Installation with all patterns selected for installation to check for
+    potential package conflicts, how the system handles big space usage, etc.
+    allpatterns installations can take longer, especially on non-x86_64
+    architectures.
```

Comment:
There is information lost. Please, don't remove

- "modules without post-installation test"
- "it doesn't publish qcow2 image."
- "Exclude modules not designed to work with all other modules together"

Reply:
I'll add back all, but the last line, as it doesn't make sense.